Content.Client/Health/BodySystem/Surgery/GenericSurgeryWindow.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.Maths;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace Content.Client.BodySystem
+{
+    public class GenericSurgeryWindow : SS14Window
+    {
+        public delegate void CloseCallback();
+        public delegate void OptionSelectedCallback(object selectedOptionData);
+
+        private Control _vSplitContainer;
+        private VBoxContainer _optionsBox;
+        private OptionSelectedCallback _optionSelectedCallback;
+
+
+        protected override Vector2? CustomSize => (300, 400);
+
+        public GenericSurgeryWindow(OptionSelectedCallback optionSelectedCallback)
+        {
+            Title = "Select surgery target...";
```

Comment:
localise.
```Loc.GetString("Select surgery target...");```

Reply:
Localized all possible strings. One thing of interest is that I didn't localize the names of the limbs, i.e. a limb might be named something in YAML like "Human Arm". Sounds like a pain for a translator to look through all the YAML's and translate them all, so I decided to hold off for now?

Reply:
I wouldn't worry about localising those yet.
The only things in yaml that we have localised are some tools, in dutch.

https://github.com/space-wizards/space-station-14/blob/master/Resources/Locale/nl-NL/tools.yml

Presumably it'd be as simple as telling the localisation it can look at limb definitions, and then having a limbs/organs localisation file with, eg:

```YAML
#../Locale/ja-JP/limbs.yml
- msgid: Hand
  msgstr: 手
```
```YAML
#../Locale/de-DE/limbs.yml
- msgid: Head
  msgstr: Kopf
```
#../Locale/ru-RU/limbs.yml
```YAML
- msgid: Leg
  msgstr: ножка
```

tl;dr, I wouldn't bother yet.